README.md

```diff
@@ -1,8 +1,119 @@
 # SQLair
 
-Friendly type mapping for SQL databases.
+SQLair is a package for Go that acts as a compatibility layer between Go and SQL databases. It allows for easy mapping between Go objects and query arguments and results.
+
+SQLair allows you to write SQL with SQLair input/output expressions included where mapping is needed. These expressions indicate the Go objects to map the database arguments/results into. SQLair is not an ORM and will leave all parts of the query outside of the expressions untouched.
+
+The API can be found at [pkg.go.dev](https://pkg.go.dev/github.com/canonical/sqlair).
+A demo can be found at [demo/demo.go](demo/demo.go).
+
+There will also soon be a full tutorial but that is currently a work in progress.
+### Motivation
+When writing an SQL query with `database/sql` in Go there are multiple points of redundency/failure:
+
+- The order of the columns in the Query must match the order of columns in `Scan`
```

Comment:
I would prefer Query to either be lowercase or to be `db.Query` to express it as a call

Reply:
Changed to lower case, I think that fits better here since we're referring to it quite generally.